barretenberg/bootstrap.sh

```diff
@@ -42,6 +42,15 @@ case "$cmd" in
   bench)
     bench
     ;;
+  bootstrap_e2e_hack)
+    echo "WARNING: This assumes you have built barretenberg and the rest of the repository is unchanged from master."
```

Comment:
Should this script also just build bberg by default?

Reply:
Yeah probably more useful that way. I was thinking people could do partial builds, but you can already just change bb binary and rerun e2e so that eats into that use-case